src/optimizer/plugins/SimplifyInstanceNameOptimizer.ts

```diff
@@ -2,11 +2,13 @@ import { OptimizerPlugin } from '../OptimizerPlugin';
 import { Package } from '../../processor';
 import { ExportableAssignmentRule } from '../../exportable';
 import ResolveInstanceOfURLsOptimizer from './ResolveInstanceOfURLsOptimizer';
+import SimplifyArrayIndexingOptimizer from './SimplifyArrayIndexingOptimizer';
 
 export default {
   name: 'simplify_instance_names',
   description:
     'Simplify Instance names by removing the appended InstanceOf information from the name (where possible)',
+  runBefore: [SimplifyArrayIndexingOptimizer.name],
```

Comment:
Why?  Is there any real interdependence between these two optimizers? As far as I can tell I can't see how the output of this optimizer would affect the operation of the `SimplifyArrayIndexingOptimizer`... but maybe I am missing something.